### DISCUSSION TITLE
Expand minio storage size

### DISCUSSION THREAD

**Discussion by chang-cancan** [minio/minio#17549] [Q&A] [answered]
Plan to use 2 server and 4 drives for each to build storage, space usage is set 50%, if few months later get another 2 server and 4 drives for each, does any chance expand to 4 server and 4 drives for each and space usage become 75%? 
If not, could you give some suggestion.
thanks!

**Comment by klauspost** [comment, marked as answer]
No. Once a cluster is deployed, the topology of these cannot be changed. This is the reason we recommend provisioning for 2 years of expected usage.

For production grade setup we recommend 4 servers with 4 drives each.

**Reply by chang-cancan** [reply]
so you mean, base on the pool, if use 2 server + 4 drives and 50% space in one pool, expand size only can add new pool and same hardware?
that a big problem for my understand.
Thank you for your answer.

**Reply by klauspost** [reply]
You can add anything that is a multiple of the original as a second pool - but it will retain the availability of the first pool, otherwise you would never know what your availability is.

It is covered here: https://min.io/docs/minio/linux/operations/install-deploy-manage/expand-minio-deployment.html

**Reply by chang-cancan** [reply]
that's mean if set storage usage 50%, then second even 3 or 4 pool still only can use 50%?

If have manay low performance server and small disk and two hight performance server and  large disk, how do you suggest to build the first pool and storage usage can set 75%? can I use low performance server to build 4 server * 4 drive in a pool, when get four high performance server, use them to build second pool and Decommission first pool?

**Reply by harshavardhana** [reply]
For more detailed discussion please reach out to us at hello@minio.io
